CHANGELOG.md

```diff
@@ -4,7 +4,53 @@
 
 #### Changed
 
-- Internal CPU mining now injects nonce bytes at the end.
+## 1.7 (2019-03-26)
+
+This version replaces all previous versions. Any prior version will stop working
+on **2020-04-02T00:00:00Z**. Node administrators must upgrade to this version
+before that date.
+
+This version will stop working on **2020-05-30T00:00:00Z**.
+
+Breaking validation changes:
+
+* Compute epoch time based on parent header. This change won't affect any
+  any users of Chainweb. The change will become effective at block height
+  452820. (#977)
```

Comment:
Estimate the date here?

Reply:
I don't think anybody cares. And it would be only an estimate.